app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SearchResultViewModel.java

```diff
@@ -136,22 +151,36 @@ public boolean getIsMySession() {
         return isMySession;
     }
 
+    public int getSearchResultId() {
```

Comment:
👍 
I think it's better to add `searchResultId` field and set this value in constructor for better performance.